marinerescue-frontend/src/pages/HomePage.js

```diff
@@ -1,14 +1,137 @@
+                                <img id="icon" src="images/beach-safety-icon.png"></img>
+                                <h2>Beach Safety</h2>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="section">
+                    <div class="section-3">
+                        <h1>Meet the Marine Rescue Team</h1>
+                        <h2>Get to know each marine animal's unique personality and story!</h2>
+                        <div class="row">
+                            <div class="column">
+                                <div class="content">
+                                    <img id="icon" src="images/strawberry.png"></img>
+                                    <h3>Strawberry</h3><br />
+                                </div>                            
+                            </div>
+                            <div class="column">
+                                <div class="content">
+                                    <img id="icon" src="images/humphrey.png"></img>
+                                    <h3>Humphrey</h3><br />
+                                </div>                            
+                            </div>
+                            <div class="column">
+                                <div class="content">
+                                    <img id="icon" src="images/sqwacky.png"></img>
+                                    <h3>Sqwacky</h3><br />
+                                </div>                            
+                            </div>
+                            <div class="column">
+                                <div class="content">
+                                    <img id="icon" src="images/jade.png"></img>
+                                    <h3>Jade</h3><br />
+                                </div>                            
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="section">
+                    <div class="section-4">
+                        <img src="images/two-jade-parents-background.png"></img>
+                        <button id="parents-button"><h1>Parents of Marine Rescuers</h1>Read more about our team and mission in educating young scientists through Marine Rescue, a citizen science project teaching debris’ impacts on marine wildlife. </button>
+                    </div>
+                </div>
+
+                <div class="section">
+                    <div class="footer">
+                    <div class="row">
+                            <div class="column">                        
+                            </div>
+                            <div class="column">
+                                <h3>Connect with us: </h3>
+                                <img id="icon" src="images/youtube-logo.png"></img> <br />
```

Comment:
```suggestion
                                <a href="https://www.youtube.com/playlist?list=PL6hj7Z2kWmbMsJqcc2mwaorQWMsKkRc9G" target="_blank"><img  src="images/youtube-logo.png"></img></a> <br />
```
Should this be a link?